docs/gethue/content/en/posts/2021-05-26-importer-direct-upload.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Importer direct upload
+author: Hue Team
+type: post
+date: 2021-05-26T00:00:00+00:00
+url: /blog/2021-05-26-importer-direct-upload
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.10
+  - Development
+
+---
+
+If you’ve ever struggled with creating new SQL tables from CSV files, you’ll be happy to learn that this is now much easier.
+
+If you know the history of Hue then you must be familiar with the importer option in Hue. So basically earlier we could create a table from a file in different databases but there is one necessary condition, the file must be present in the filesystem storage like [HDFS](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#Introduction) or [s3](https://gethue.com/introducing-s3-support-in-hue/). Now we can create a table from local storage files also in different [databases](https://docs.gethue.com/administrator/configuration/connectors/).
+
+### Goal:
+
+* Propose a direct upload of small file which does not depend on a storage system in the importer.
+
+### Why:
+
+* If someone doesn't have HDFS or s3 configured in Hue then it is difficult to create the table from a file, so if we have the feature like this then anyone can use this feature for creation of tables. And ofcourse, it also saves time as it directly uploads the file to the importer.
+
+### Steps to create a table:
+
+![Importer direct upload steps gif](https://cdn.gethue.com/uploads/2021/05/Importer_direct_upload_steps.gif)
+
+### Describing the steps through the app diagram:
+![Importer direct upload steps app diagram](https://cdn.gethue.com/uploads/2021/05/Importer_direct_upload_app_diagram.png)
```

Comment:
The blocks in the diagram look a bit inconsistent.
Could you please redo it? Suggestion:
Select a file --> Select a dialect (Hive, Impala, Phoenix, MySQL, ...) --> Review/Edit column names and data type --> Submit

Put the text in the boxes and keep the arrows blank.